### PR TITLE
Init command improvements

### PR DESCRIPTION
**Pull request by Nashati**

### CODE REVIEW

File: src/cmds/init.cmd.js [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
 export const command = 'init';
 
 export const desc = 'init Perfecto and Cypress configuration files';
 
 export const builder = {
   prompt: { default: true, hidden: true, boolean: true },
-  'tests.path': {...testsOptions['tests.path']}
+  'tests.path': { ...testsOptions['tests.path']},
+  ...credentialsOptions,
+  skip: { alias: 's', type: 'boolean', default: false, describe: 'Skip interactive questions' },
```
Comment (tehilaAzrad) [inline src/cmds/init.cmd.js:13]: maybe the alias should be skip and not s? s it's too generic and  I think we use this already for security token

File: src/cmds/init.cmd.js [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 };
 
 const getInitOptions = (argv) => {
   return {
-    ...(!argv?.tests?.path) && {testsPath : { type: 'input', describe: 'Enter your tests path' }},
-    cloud: { type: 'input',describe: 'Enter your Perfecto cloud name' },
-    projectName: { type: 'input', describe: 'Enter a project name for your test root folder (only if there is no package.json in this folder)' },
```
Comment (tehilaAzrad) [inline src/cmds/init.cmd.js:18]: why you delete the question about project name ?

**Review by tehilaAzrad — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test-release-workflow.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         node-version: [10.x.x, 12.x, 14.x]
-        os: [ubuntu-latest, windows-latest, macos-latest]
```
[the remark this answers is not in the document]
Comment (Nashati) [reply .github/workflows/build-test-release-workflow.yml:10]: There was an issue in Github with running on macs, I will readd this

File: src/cmds/init.cmd.js (hunk 2 above)
Comment (Nashati) [reply src/cmds/init.cmd.js:18]: Project name was used in `package.json` which we don't add anymore

File: src/cmds/init.cmd.js (hunk 1 above)
Comment (Nashati) [reply src/cmds/init.cmd.js]: I disabled the alias

**Review by tehilaAzrad — APPROVED** [review]

**Pull request merged**